columnq-cli/src/main.rs

```diff
@@ -31,13 +31,12 @@ fn config_path() -> anyhow::Result<PathBuf> {
     Ok(home)
 }
 
-fn table_arg() -> clap::Arg<'static> {
+fn table_arg() -> clap::Arg {
     clap::Arg::new("table")
         .long_help("Table sources to load. Table option can be provided as optional setting as part of the table URI, for example: `blogs=s3://bucket/key,format=delta`. Set table uri to `stdin` if you want to consume table data from stdin as part of a UNIX pipe. If no table_name is provided, a table name will be derived from the filename in URI.")
-        .takes_value(true)
+        .num_args(1)
```

Comment:
I'm not sure if this was intended, but this seems to have made the option only usable once - you can't pass `--table` multiple times for `columnq console`.` The `README.md` includes an example with multiple `--table` arguments, and that seems unsupported:

```
error: the argument '--table <[table_name=]uri[,option_key=option_value]>' cannot be used multiple times

Usage: columnq console [OPTIONS]

For more information, try '--help'.
```

Reply:
Yeah, installing 51584dd3a09f7600827844e4357f9f6915a34155 fixes the issue.